tl/utility/utility.py

```diff
@@ -229,54 +245,117 @@ def to_float(input_str):
             return None
 
     @staticmethod
-    def create_mapping_es(str_fields_need_index: typing.List[str], str_fields_no_index: typing.List[str] = None):
+    def create_mapping_es(es_version, str_fields_need_index: typing.List[str], float_fields: typing.List[str] = None, 
```

Comment:
add typing for es_version like other parameters

Reply:
Done